content/applications/general/companies/digest_emails.rst

```diff
@@ -169,9 +176,8 @@ Click :guilabel:`Save` to implement the changes.
 Create digest emails
```

Comment:
A majority of this section is just duplicate content from what's on line 38. Wondering if the two can be combined so it's one solid section early in the doc?

Reply:
@StraubCreative I updated the PR based on the comments except this one. Don't feel like I know enough about the writing style to combine these sections.